MetaMask/metamask-extension
# fix: settings items don't wrap lines when there is enough space

## Explanation



* What is the current state of things and why does it need to change?

In the settings page, the text of many items is forced to be line wrap when there is plenty of space on the right side.

<img width="402" alt="截屏2023-06-09 上午1 38 16" src="https://github.com/MetaMask/metamask-extension/assets/45708948/0c455718-7c29-4053-a327-736876caf817">


It happens not only in Chinese, but also in many other languages (even English).

It destroys the integrity of the text and makes it more difficult and unattractive to read.

* What is the solution your changes offer and how does it work?

Open chrome DevTool to inspect. We can see that the space of the parent(`tab-bar__tab__content`) of the text element(`tab-bar__tab__content__title`) is not full.

<img width="636" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/45708948/603ec21d-4ffe-4723-818c-111d4248a726">

<img width="639" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/45708948/03653efe-23d9-426f-9f33-dab5ad930b1c">

In that case, add `width: 100%` syle to this class. Just make it take up the full width.

## Screenshots/Screencaps



<table>
  <tr>
    <th>Before</th>
    <th>After</th>
  </tr>

  <tr>
    <td>
<img width="373" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/45708948/b75456fe-ef85-463f-91a5-c81cc65dcd0f">
</td>
    <td><img width="391" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/45708948/fbec9866-4b16-4a2d-b2fd-e6196b5b73b4">
</td>
  </tr>

  <tr>
    <td>
<img width="393" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/45708948/ba39123e-5994-4d1c-b86e-15922270f0fa">

</td>
    <td>
<img width="381" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/45708948/152ba7b9-f6aa-46d9-9ff6-e7149a55d282">

</td>
  </tr>

 <tr>
    <td>
<img width="382" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/45708948/54940f64-43d5-4b80-8b2c-d0020b8a6ba0">

</td>
    <td>
<img width="382" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/45708948/894b89d0-994c-4a73-b7ee-b2497890d01c">

</td>
  </tr>

 <tr>
    <td>
<img width="379" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/45708948/ed333e01-6bdb-4d39-a231-bd2cbad513d5">

</td>
    <td>
<img width="387" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/45708948/3fde350c-87e0-4e53-9057-120c91d39c8a">

</td>
  </tr>
</table>

## Manual Testing Steps



1. Open Settings Page
2. If you want to see the changes in other languages, go to Settings - General page and change the Current Language

## Pre-merge author checklist

  - [] How reviewers can test my changes

## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
